articles/hosted-pages/login/index.md

```diff
@@ -0,0 +1,127 @@
+
+### How Does the Hosted Login Page Work
+
+Auth0 shows the Hosted Login Page whenever something (or someone) triggers an authentication request, such as calling the `/authorize` endpoint (OIDC/OAuth) or sending a SAML login request. Users will see the Hosted Login Page, typically with either the Lock widget or with your custom UI. Once they login, they will be redirected back to your application.
+
+#### Social Providers
+
+If the incoming authentication request includes a `connection` parameter that uses an external identity provider (such as a social provider), the Hosted Login Page will not display. Instead, Auth0 will direct the user to the identity provider's login page.
+
+#### SSO
+
+When SSO is desired, a client should use the Hosted Login Page for logins rather than an embedded login solution. When a user logs in via the Hosted Login Page, a cookie will be created and stored. On future calls to the `authorize` endpoint, the cookie will be checked, and if SSO is achieved, the user may not ever be redirected to the Hosted Login Page. They will see the page only when they need to actually login. 
+
+This behavior occurs without modification to the actual Hosted Login Page. The process involves enabling SSO for the client in the [Dashboard](${manage_url}) and then using the `authorize` endpoint with `?prompt=none` for silent SSO.
+
+For more information about how SSO works, see the [SSO documentation](/sso).
+
+![Hosted Login Page Preview](/media/articles/hosted-pages/hlp-preview-lock.png)
+
+### What the Hosted Login Page is Not Intended For
+
+An important thing to remember is that the Hosted Login Page is intended to be only for signups and authentication. Attempting to host any significant amount of application logic in the Hosted Login Page is not advised.
+
+The Hosted Login Page is truly a single page constructed within the editor, so all custom styling and includes will need to be put into the single editor window. Hosting other files or images along with the Hosted Login Page is not possible, so those will need to be hosted in the application itself, or elsewhere.
+
+### Hosted Login vs Embedded Login
+
+Why use the Hosted Login Page rather than embedding your login functionality within your application?
+
+Security is the primary reason, followed by ease of setup. Cross-origin authentication is inherently more dangerous, and more likely to be vulnerable to man-in-the-middle attacks. Using the Hosted Login Page for the authentication process with Auth0 prevents that from ever being a concern. Additionally, the Hosted Login Page is very easy to implement, especially if a custom UI is not required.
+
+### Passwordless on Native Platforms
+
+Currently, the Hosted Login Page is the _only_ way to use Passwordless authentication on Native platforms. So if your use case is a native iOS or Android application, for example, and you intend to implement Passwordless, you will need to use the Hosted Login Page.
```

Comment:
The italics thing annoys me as I'm used to using them when syntactically appropriate in writing. Sorry haha